flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateSerializerImplTest.java

```diff
@@ -146,4 +145,10 @@ private void readAndCheck(byte[] data, ChannelStateSerializerImpl serializer, By
 		}
 	}
 
+	static byte[] generateData(int len) {
```

Comment:
What's that change about?

Reply:
Previously, this method was imported from `ChannelStateReaderImplTest`.
In this commit `ChannelStateReaderImplTest` was removed and `generateData` moved to this class.